### ISSUE TITLE
TypeError when running the example program

### ISSUE THREAD

**Issue by git-commit** [Pytwitcher/pytwitcherapi#16]
I am currently running python 3.4.0.

Stacktrace:
```
Type in the channel to join:test
Traceback (most recent call last):
  File "donavader.py", line 64, in <module>
    main()
  File "donavader.py", line 49, in main
    client = create_client(session)
  File "donavader.py", line 43, in create_client
    return IRCClient(session, channel)
  File "c:\Python34\lib\site-packages\pytwitcherapi\chat\client.py", line 244, in __init__
    self.channel = channel
  File "c:\Python34\lib\site-packages\pytwitcherapi\chat\client.py", line 307, in channel
    ip, port = self.session.get_chat_server(channel)
  File "c:\Python34\lib\site-packages\pytwitcherapi\session.py", line 575, in get_chat_server
    stats = [client.ChatServerStatus(**d) for d in r.json()]
  File "c:\Python34\lib\site-packages\pytwitcherapi\session.py", line 575, in <listcomp>
    stats = [client.ChatServerStatus(**d) for d in r.json()]
TypeError: __init__() got an unexpected keyword argument 'pings'
```

**Comment by storax** [comment]
hey sorry, i currently don't have time to actively maintain this. feel free to create a pull request and i will review and merge it. This package hasn't had an update in a while and the twitch api moved on.
I will update the readme, sorry for that.

**Comment by storax** [comment]
I found the time to fix this and a couple other issues and released 0.9.1.
The example chat client has been tested with Python 3.4.
Thank you for reporting this ticket.
Sorry, I haven't done this earlier but this project has very low priority at the moment.
I hope you still find this useful. I'm going to close this ticket. If anything comes up, feel free to open another one or submit a PR.

Cheerio,
storax
